test/e2e/autoscale.go

```diff
@@ -410,3 +400,25 @@ func AutoscaleUpToNumPods(ctx *TestContext, curPods, targetPods float64, done <-
 
 	return grp.Wait
 }
+
+// isInRollout is a loose copy of the kubectl function handling rollouts.
+// See: https://github.com/kubernetes/kubectl/blob/0149779a03735a5d483115ca4220a7b6c861430c/pkg/polymorphichelpers/rollout_status.go#L75-L91
+func isInRollout(deploy *appsv1.Deployment) bool {
+	if deploy.Generation > deploy.Status.ObservedGeneration {
```

Comment:
or 
return a || b || c&&d ... :)

Reply:
I contemplated that but... sometimes verbosity wins :D